src/ConditionsResolver/Extensions/RegexExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using System.Text.RegularExpressions;
+
+namespace Monai.Deploy.WorkflowManager.ConditionsResolver.Extensions
+{
+    public static class RegexExtensions
+    {
+        public static string[] SplitOnce(this Regex regex, string input)
+        {
+            var inputArr = regex.Split(input);
```

Comment:
Just FYI this call may [throw ](https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.split?view=net-6.0#system-text-regularexpressions-regex-split(system-string)) if input is null